test/test_helpers.py

```diff
@@ -55,7 +55,7 @@ def cleanup():
     # TODO: this hardcodes the path to bash.exe, which works for now but might
     #       need to be set via environ or something like that.
     path.with_suffix(".bat").write_text(
-        '@"\\Program Files\\git\\bin\\bash.exe" {} %*\n'.format(path))
+        '@"C:\\Program Files\\git\\bin\\bash.exe" {} %*\n'.format(path))
```

Comment:
This change was needed because on GitHub Actions, the code is run from `D:` so need to hard-code `C:` here.

Reply:
This definitely feels like something that should be dealt with via PATH, etc, as assuming C:\Program Files\git seems a bit weak. But it does work for actions/travis.